README.md

```diff
@@ -1,10 +1,10 @@
-# Portfolio
+# Capstone
 
-> Welcome to my HTML + CSS + Javascript Capstone project, feel free to look around.
+> Welcome to my HTML + CSS + Javascript Capstone project, feel free to look around. For this project I created a concert for developers sponsored by Microverse.
 
-- Live version: [GitHub Page](https://github.com/smunozmo/HTML-basic-JavaScript-Capstone)
+- Live version: [GitHub Page](https://smunozmo.github.io/Module_1_Capstone/)
 
-![screenshot](./)
+![screenshot](./assets/img/screenshot.png)
 
 ## Built With
```

Comment:
- [ ] Add more information on how to run the project locally, keeping in mind that there might be non-technical users trying to run it.  - You need to make the change that the last reviewer request, go back and check the README template, there's a section "Getting Started", a professional repository is super important for future projects and jobs.

Reply:
Thank you @mricanho 🙌